### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by jstloyal**
- Working with Form by building a rials app call re-former
- Create a User model
- Add HTML form in the beginning
- And using the helper method rails provide

### CODE REVIEW

**Review by wathigo — CHANGES_REQUESTED** [review]
## Changes Required 
Great work :clap: 
I was able to create a user successfully.
Let's handle some requirements before we move on.

### REQUIRED
**Views**
* Contains a commented out form ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) from Odin).
* Contains a commented out form_tag ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from Odin).

**After working on the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<%= form_tag('/users', method: :post) do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %><br><br>
+  <%= label_tag(:email, "Email:") %>
+  <%= email_field_tag(:email) %><br><br>
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag(:password) %><br><br>
+  <%= submit_tag("Create") %>
+<% end %>
+-->
```
Comment (kobiyoyo) [inline app/views/users/new.html.erb:25]: You can write it like this 
````ruby
<% 
=begin %>
<%= form_tag('/users', method: :post) do %>
  <%= label_tag(:username, "Username:") %>
  <%= text_field_tag(:username) %><br><br>
  <%= label_tag(:email, "Email:") %>
  <%= email_field_tag(:email) %><br><br>
  <%= label_tag(:password, "Password:") %>
  <%= password_field_tag(:password) %><br><br>
 <%= submit_tag("Create") %>
<% end %>
<% 
=end %>

````

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team,  You have done a great job with this project. :clap:

#### You can go ahead and merge !!!

File: app/views/users/new.html.erb (hunk 1 above)
Comment (jstloyal) [reply app/views/users/new.html.erb:25]: Okay, I will take note of that.

**Pull request merged**